server/api/analytics/db/dbQueries.js

```diff
@@ -0,0 +1,31 @@
+const db = require('./dbConnection');
+const qry = require('./dbQryStrs');
+
+// get all results in DB
+exports.getAllResults = (cb) => {
+  db.query(qry.getAllResults,
+  (error, result) => {
+    if (error) {
+      console.error(error);
```

Comment:
I don't think we can just console.error here, b/c then the client is left hanging. I think what makes the most sense is to assume that the callback will take (err, result), and call it with (err, null) if there's an error.